modules/openapi-generator/src/main/java/org/openapitools/codegen/utils/ModelUtils.java

```diff
@@ -1126,7 +1134,10 @@ public static String getParentName(ComposedSchema composedSchema, Map<String, Sc
                     } else {
                         // not a parent since discriminator.propertyName is not set
                         hasAmbiguousParents = true;
-                        refedWithoutDiscriminator.add(parentName);
+                        Boolean isNotExtractedInlineSchmea = !parentName.equals(schemaName+"_allOf");
```

Comment:
Typo: Schmea.

Reply:
Thanks for the catch. This typo has been corrected.